docs/kickstart-docs.rst

```diff
@@ -83,6 +83,9 @@ kickstart file:
    user would be prompted during a typical installation. Once the answer
    is given, the installation will continue unattended unless it finds
    another missing item.
+-  Commands ``url``, ``nfs``, ``cdrom`` or ``harddrive`` are required for
```

Comment:
Also `liveimg` and `hmc`. In general, every command specified in the `method` command. 

Maybe the list of commands could be generated in the `method`'s help and here it could be just pointed out that we require one of the installation methods specified at `method`

Reply:
Hmm interesting idea. I'll look on that.